python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -262,6 +263,28 @@ def _get_ray_resources_from_group_spec(
     if num_gpus is not None:
         resources["GPU"] = num_gpus
 
+    if num_tpus is not None:
+        """Add TPU head resource, similar to the GCP node_provider.
```

Comment:
We need to update the comment after https://github.com/ray-project/ray/pull/45105/files#r1626459384.

Reply:
Done in [ec92b79](https://github.com/ray-project/ray/pull/45105/commits/ec92b791e9d706b0986c9132cd0d82855346b5cd).